### PR TITLE
Add ESLint CI

### PR DESCRIPTION
**Pull request by oddstr13**
This adds Reviewdog automated reviews for ESLint problems

`npm install -D` sorted packages by name - sorry about that mess.
ESLint 9 forces new config format.

### CODE REVIEW

**Comment by oddstr13** [comment]
Just merge if review passes :smile_cat:

File: registration-app/package.json [hunk 1]
```diff
@@ -2,25 +2,30 @@
...
     "build": "next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
+    "next": "15.0.2",
     "react": "19.0.0-rc-02c0e824-20241028",
-    "react-dom": "19.0.0-rc-02c0e824-20241028",
-    "next": "15.0.2"
+    "react-dom": "19.0.0-rc-02c0e824-20241028"
```
Comment (WindShiba) [inline registration-app/package.json:17]: Afaik React v19 is considered an unstable version. Rena mentioned something about this. Should we continue with it, or would it be better to go back for a stable release? imo we keep running with v19.

Comment (oddstr13) [reply registration-app/package.json:17]: rc stands for release candidate - usually comes between the last beta and before v19.0.0, but I'm not familiar with react release policy and process.

Either way, this PR doesn't change the react version, it only adds a few dependencies for ESLint to work properly. npm sorted the dependencies when I used the command to add them, hence the mess here.

File: registration-app/app/page.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import Image from "next/image";
+//import Image from "next/image";
```
Comment (wiwalla) [inline registration-app/app/page.tsx]: can just be deleted, no need for it staying there 👍🏻

**Review by oddstr13 — COMMENTED** [review]
> can just be deleted, no need for it staying there 👍🏻  

Only reason for the change was to satisfy ESLint – I figured the whole file would be changed many times over anyways, so I just did a quick comment.

Deleted now.

**Pull request merged**